app/Http/Controllers/BookTitleController.php

```diff
@@ -28,9 +30,57 @@ public function index(Request $request)
             });
         }
 
-        $bookTitles = $filtered->with('genres')->paginate(BookTitle::PAGINATE);
+        $filtered = $filtered->with('genres')->paginate(BookTitle::PAGINATE);
+
+        $bookTitles = $filtered->getCollection()->transform(function ($title) {
+            $title->genres = $title->genres->pluck('name')->join(', ');
+            $title->released_date = date("m/Y", strtotime($title->released_date));
```

Comment:
Em tìm hiểu về accessor trong Laravel: https://laravel.com/docs/8.x/eloquent-mutators#accessors-and-mutators